src/circuit/gadget/sinsemilla.rs

```diff
@@ -222,6 +226,7 @@ pub struct HashDomain<
     EccChip: EccInstructions<
             C,
             Point = <SinsemillaChip as SinsemillaInstructions<C, K, MAX_WORDS>>::Point,
+            FixedPoints = <SinsemillaChip as SinsemillaInstructions<C, K, MAX_WORDS>>::FixedPoints,
```

Comment:
Do we want these to be identical? I agree that `EccChip::FixedPoints` needs to contain all fixed points we want to use with `CommitDomain`, but ideally it could be a superset.

Non-blocking because we do have the `CommitDomains` trait which means someone can't accidentally use a non-SinsemillaCommit fixed point, and I can imagine the type restrictions to make this a superset could get somewhat ugly.